src/Components/Todo.jsx

```diff
@@ -1,23 +1,42 @@
 import React, { Fragment } from "react";
 import { useDispatch } from "react-redux";
 import PropTypes from "prop-types";
-import { deleteTodo } from "services/actions/todo";
-import { ICON_DELETE, ALT_TEXT_DELETE } from "Common/constants";
+import { deleteTodo, markDone } from "services/actions/todo";
+import {
+  ICON_DELETE,
+  ICON_CHECK,
+  ALT_TEXT_DELETE,
+  ALT_TEXT_CHECK,
+} from "Common/constants";
 import Button from "./Button";
 
-export default function Todo({ id, text, createdAt }) {
+export default function Todo({ id, text, createdAt, isCompleted }) {
   const dispatch = useDispatch();
 
   const handleDelete = () => {
     dispatch(deleteTodo(id));
   };
 
+  const handleDone = () => {
+    console.log(isCompleted)
+    if (!isCompleted) {
+      dispatch(markDone(id));
+    }
+  };
+
   return (
     <Fragment>
-      <h1>{text}</h1>
+      <h1 style={isCompleted ? { textDecoration: "line-through", color: "green" } : {}}>
```

Comment:
```suggestion
      <h1 style={isCompleted && { textDecoration: "line-through", color: "green" } }>
```
can we use it like this?